scripts/install-conda.sh

```diff
@@ -1,25 +1,9 @@
 #!/usr/bin/env bash
 set -xe
-# Requires environment CONDA_SHA256, CONDA_VERSION, and DEFAULT_ENV
-wget --quiet -O miniconda.sh https://repo.anaconda.com/miniconda/Miniconda3-$CONDA_VERSION-Linux-x86_64.sh
-
-echo "${CONDA_SHA256} miniconda.sh" > miniconda.checksum
-
-if [ $(sha256sum -c miniconda.checksum | awk '{print $2}') != "OK" ]; then
-   echo Error when testing checksum
-   exit 1;
-fi
 
+curl -sL -o miniconda.sh "https://github.com/conda-forge/miniforge/releases/latest/download/Mambaforge-$(uname)-$(uname -m).sh"
```

Comment:
In previous builds, we are pinning the version using `CONDA_VERSION` (inside of the Dockerfile.jupyterlab), perhaps its worthwhile pinning the version here as well. In the original issue, Tania also recommended that we upgrade to python 3.8.

Reply:
The recommendation to upgrade to Python 3.8 was to be done in a different PR if I remember correctly.